### ISSUE TITLE
How to export/extract out project created in desktop 1.3.9 ver as sjr file to local?

### ISSUE THREAD

**Issue by madusikat** [jfo8000/ScratchJr-Desktop#36]
Hi, can help guide how to export the projects created with Desktop 1.3.9 ver to local desktop? I like to email the created projects to share out to Teachers for review. Thanks in advance for the help!

**Comment by klenz39** [comment]
I dont think you can share individual project in desktop version. You may send your whole saved file by going to Document/ScratchJr and copy scratchjr.sqllite . Then put it in the same place in different computer

**Comment by mink90778** [comment]
just hack it...

**Comment by mink90778** [comment]
WAIT NO JUST NAGIVATE THE FILE
